packages/swarm-docs/src/components/layout.js

```diff
@@ -37,23 +46,26 @@ const Layout = ({ children }) => {
             <html lang="en" />
             {require('swarm-icons/dist/sprite/sprite.inc')}
           </Helmet>
-          <Header siteTitle={data.site.siteMetadata.title} />
-          <div
-            style={{
-              margin: '0 auto',
-              maxWidth: 960,
-              padding: '0px 1.0875rem 1.45rem',
-              paddingTop: 0,
-            }}
-          >
-            {children}
+          <Nav fileNames={ data.allFile.edges.map((value) => value.node.name)} />
```

Comment:
recommend changing your diff settings to more easily see the changes:
![image](https://user-images.githubusercontent.com/5428933/53511197-d5022c80-3a8d-11e9-9e18-a573e15cc6d2.png)